docs/platform-external/use-case-oci-upi.md

```diff
@@ -0,0 +1,1829 @@
+  --instance-details file://instance-config-details-compute.json
+```
+
+- Creating the Instance Pool
+
+```sh
+INSTANCE_POOL_COMPUTE="${CLUSTER_NAME}-compute"
+INSTANCE_CONFIG_ID_CMP=$(oci compute-management instance-configuration list \
+  --compartment-id $COMPARTMENT_ID_OPENSHIFT \
+  | jq -r ".data[] | select(.[\"display-name\"] | startswith(\"$INSTANCE_CONFIG_COMPUTE\")).id")
+
+BES_HTTP_NAME=$(oci nlb backend-set list --network-load-balancer-id $NLB_ID \
+  | jq -r '.data.items[] | select(.name | endswith("http")).name')
+BES_HTTPS_NAME=$(oci nlb backend-set list --network-load-balancer-id $NLB_ID \
+  | jq -r '.data.items[] | select(.name | endswith("https")).name')
+
+#
+# oci compute-management instance-pool create --generate-param-json-input load-balancers
+cat <<EOF > ./instance-pool-loadbalancers-cmp.json
+[
+  {
+    "backendSetName": "$BES_HTTP_NAME",
+    "loadBalancerId": "$NLB_ID",
+    "port": 80,
+    "vnicSelection": "PrimaryVnic"
+  },
+  {
+    "backendSetName": "$BES_HTTPS_NAME",
+    "loadBalancerId": "$NLB_ID",
+    "port": 443,
+    "vnicSelection": "PrimaryVnic"
+  }
+]
+EOF
+
+oci compute-management instance-pool create \
+  --compartment-id $COMPARTMENT_ID_OPENSHIFT \
+  --instance-configuration-id "$INSTANCE_CONFIG_ID_CMP" \
+  --size 0 \
+  --display-name "$INSTANCE_POOL_COMPUTE" \
+  --placement-configurations "[{\"availabilityDomain\":\"$AVAILABILITY_DOMAIN\",\"faultDomains\":[\"FAULT-DOMAIN-1\",\"FAULT-DOMAIN-2\",\"FAULT-DOMAIN-3\"],\"primarySubnetId\":\"$SUBNET_ID_PRIVATE\"}]" \
+  --load-balancers file://instance-pool-loadbalancers-cmp.json
+```
+
+- Scale up
+
+```sh
+INSTANCE_POOL_ID_CMP=$(oci compute-management instance-pool list \
+    --compartment-id  $COMPARTMENT_ID_OPENSHIFT \
+    | jq -r ".data[] | select(
+        (.[\"display-name\"]==\"$INSTANCE_POOL_COMPUTE\") and
+        (.[\"lifecycle-state\"]==\"RUNNING\")
+    ).id")
+
+oci compute-management instance-pool update --instance-pool-id $INSTANCE_POOL_ID_CMP --size 2
+```
+
+### Review the installation
+
+Export the kubeconfig
+
+```sh
+export KUBECONFIG=$INSTALL_DIR/auth/kubeconfig
+```
+
+#### OCI Cloud Controller Manager
+
+- Check if the CCM pods have been started and nodes initialized:
+
+```sh
+oc logs -f daemonset.apps/oci-cloud-controller-manager -n oci-cloud-controller-manager
+```
+
+Example output:
+
+```
+I0816 04:22:12.019529       1 node_controller.go:484] Successfully initialized node inst-rdlw6-demo-oci-003-controlplane.priv.ocp.oraclevcn.com with cloud provider
+```
+
+- Check if the nodes have been initialized
+
+```sh
+oc get nodes
+```
+
+- Check if the controllers are running for each master:
+
+```sh
+oc get all -n oci-cloud-controller-manager
+```
+
+#### Approve certificates for worker nodes
+
+Once the instances are created by the provider and ignitions loaded, the certification signing requests (CSR's) must be created by kubelet awaiting to be approved - in pending condition.
```

Comment:
this part is a little confusing to me

> the certification signing requests (CSR's) must be created by kubelet awaiting to be approved - in pending condition.

it sounds like the user needs to create the requests in pending state, but i thought the kubelet creates the requests?